strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/test/java/org/carlspring/strongbox/util/PypiMetadataParserTest.java

```diff
@@ -5,12 +5,19 @@
 import org.carlspring.strongbox.domain.PypiPackageInfo;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.ContextConfiguration;
 
+import javax.validation.ConstraintViolationException;
 import java.io.FileInputStream;
 import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
 
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
We decided to switch from junit and hamcrest to AssertJ.
Please change assertions to AssertJ way.
See: https://github.com/strongbox/strongbox/issues/1456

Reply:
@mslowiak Test cases are updated to use AssertJ fluent assertions. Thanks!

Reply:
@sainalshah thank you for your follow up, good job :)